firestore/src/android/exception_android.cc

```diff
@@ -1,5 +1,8 @@
 #include "firestore/src/android/exception_android.h"
 
+#include <stdexcept>
```

Comment:
Does this build on stlport? (You can run CPP packaging on this PR to check.)

Reply:
Yep, I [ran a packaging workflow which failed](https://github.com/firebase/firebase-cpp-sdk/actions/runs/649538742) in the iOS tests, but so does the nightly build, unfortunately.  Looks like we have some mac ios scenarios to deflake.